node/pkg/driver/node_utils.go

```diff
@@ -295,6 +300,45 @@ func (n NodeUtils) RemoveFileOrDirectory(path string) error {
 	return os.RemoveAll(path)
 }
 
+func (n NodeUtils) MakeDir(dirPath string) error {
+	err := os.MkdirAll(dirPath, os.FileMode(0755))
+	if err != nil {
+		if !os.IsExist(err) {
+			return err
+		}
+	}
+	return nil
+}
+
+func (n NodeUtils) MakeFile(filePath string) error {
+	f, err := os.OpenFile(filePath, os.O_CREATE, os.FileMode(0644))
+	defer f.Close()
+	if err != nil {
+		if !os.IsExist(err) {
+			return err
+		}
+	}
+	return nil
+}
+
+func (n NodeUtils) FormatDevice(devicePath string, fsType string) {
+	var args []string
+	if fsType == "ext4" {
+		args = []string{"-m0", "-Enodiscard,lazy_itable_init=1,lazy_journal_init=1", devicePath}
+	} else if fsType == "xfs" {
+		args = []string{"-K", devicePath}
+	} else {
+		logger.Errorf("Could not format unsupported fsType: %v", fsType)
+		return
+	}
+
+	logger.Debugf("Formatting the device with fs_type = {%v}", fsType)
+	_, err := n.Executer.ExecuteWithTimeout(int(mkfsTimeout.Seconds()*1000), "mkfs."+fsType, args)
```

Comment:
if this is the only use of `mkfsTimeout` so why this is not the default?

Reply:
not the default of what? of `mSeconds` parameter in `ExecuteWithTimeout`? this function is used in many places in the project

Reply:
of `mkfsTimeout`

Reply:
`mkfsTimeout` is a const. what do you mean by "default"? and what is not the default?

Reply:
`int(mkfsTimeout.Seconds()*1000) ` is the only use of `mkfsTimeout `, why `mkfsTimeout` cant just be the result of `int(mkfsTimeout.Seconds()*1000) ` from the start?

Reply:
would `mkfsTimeoutMilliseconds = 15 * 60 * 1000` be better than the current `mkfsTimeout = 15 * time.Minute`?
if so, please confirm and I will update it accordingly

Reply:
yes, I prefer `15 * 60 * 1000`

Reply:
done